app/views/posts/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2><%= link_to "Make a New Post", new_post_path %></h2>
+
+<% (0..(@all_posts)/12).each do |page_num| %>
```

Comment:
this looks like a magic number, and it should probably be abstracted out of the view file.